pkg/acsengine/const.go

```diff
@@ -17,6 +17,8 @@ const (
 	DefaultKubernetesMasterSubnet = "10.240.0.0/16"
 	// DefaultKubernetesClusterSubnet specifies the default subnet for pods.
 	DefaultKubernetesClusterSubnet = "10.244.0.0/16"
+	// DefaultDockerBridgeSubnet specifies the default subnet for the docker bridge network for masters and agents.
+	DefaultDockerBridgeSubnet = "172.17.0.5/16"
```

Comment:
@JackQuincy if you set it to 172.17.0.0/16 with --bip, it will fail. We could set it to 172.17.0.1/16, but I'm pretty sure that 172.17.0.5 is the default when --bip is not specified.